test/AWS.Deploy.CLI.IntegrationTests/SaveCdkDeploymentProject/ReDeploymentTests/CompatibleReDeployment.cs

```diff
@@ -0,0 +1,182 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
It is not ideal to grow the integration suite at this pace. There are other ways to test (especially unit test).

At max, there should be one integration test that validates that a saved deployment project deploys successfully.

Remember test pyramid

![image](https://user-images.githubusercontent.com/8882380/134603121-16b86a17-d787-4aa7-91aa-1a8875d16520.png)

Reply:
I have streamlined the the compatible and incompatible re-deployment in a single integration test. This is the first (and the only) integration test that performs any sort of deployment using the persisted CDK project.